validator/sawtooth_validator/consensus/poet1/wait_certificate.py

```diff
@@ -120,10 +125,7 @@ def wait_certificate_from_serialized(cls,
 
     @property
     def enclave_wait_certificate(self):
-        return \
-            self.poet_enclave.deserialize_wait_certificate(
-                self._serialized_certificate,
-                self.signature)
+        return self._enclave_wait_certificate
```

Comment:
Like wise, this change seems to be unrelated to the keyword args change.  Not clear on why the change is needed.